packages/pages/src/components/link/link.tsx

```diff
@@ -23,15 +23,15 @@ interface LinkConfig
 /**
  * The shape of the data passed to {@link Link} when directly passing an HREF to the Link component.
  */
-interface HREFLinkProps extends LinkConfig {
+export interface HREFLinkProps extends LinkConfig {
```

Comment:
Why do these need to be exported? I don't see them used in the story.

Reply:
This is an [issue in storybook](https://github.com/storybookjs/storybook/issues/12064). I could also duplicate the type in the storybook file. [This](https://stackoverflow.com/a/63138461/5627731) provides some more info into the error I believe, removing declarations in tsconfig would also resolve the error